core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-navigation-bar/edit-ema-navigation-bar.component.scss

```diff
@@ -0,0 +1,60 @@
+@use "variables" as *;
+
+:host {
+    width: 80px;
+    height: 100%;
+}
+.edit-ema-nav-bar {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    background-color: $color-palette-primary-100;
+    border-left: 1px solid $color-palette-primary-300;
+
+    .edit-ema-nav-bar__item {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        height: 88px;
+        width: 100%;
+        gap: $spacing-0;
+        padding: 0 $spacing-0;
+        text-decoration: none;
+        color: $black;
+
+        .pi {
+            font-size: $font-size-xl;
+            color: $color-palette-gray-800;
+        }
+
+        .item__image {
+            -webkit-user-drag: none;
+            user-select: none;
+            -moz-user-select: none;
+            -webkit-user-select: none;
+            -ms-user-select: none;
+        }
+
+        .item__label {
+            text-align: center;
+            text-wrap: wrap;
+            font-size: $font-size-sm;
+        }
+    }
+
+    .edit-ema-nav-bar__item--active {
+        background-color: $white;
+        border: 1px solid $color-palette-primary-300;
+        border-left: 1px solid $white;
+        width: 81px; // This makes the illusion of blend with the background because it covers the border of the container
+
+        .pi,
+        .item__label {
+            color: $color-palette-primary-500;
+        }
+    }
```

Comment:
Don't need nesting... BEM does it implicitly. Actually doesn't need BEM at all... but is something I would like to talk with the whole team.

Reply:
unnested!